dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/EnrollmentAnalyticsService.java

```diff
@@ -59,7 +59,7 @@
 
     String ITEM_ORG_UNIT_CODE = "oucode";
 
-    String ITEM_ORG_UNIT_ID = "ou";
+    String ITEM_PROGRAM_STATUS = "programstatus";
```

Comment:
I do not understand, It looks like you are replacing/removing ou

Reply:
The ITEM_PROGRAM_STATUS is a new constant.
The constant ITEM_ORG_UNIT_ID was removed because was not being used anywhere in the codebase.